app/models/division.rb

```diff
@@ -114,4 +114,20 @@ def locale_names
       I18n.t("locale_name.#{locale}", locale: locale)
     end
   end
+
+  def quickbooks_connected?
+    self.class.connector.connected?
+  end
+
+  def quickbooks_connect(access_token:, params:)
+    self.class.connector.connect(access_token: access_token, params: params)
+  end
+
+  def quickbooks_disconnect
+    self.class.connector.disconnect
+  end
+
+  def self.connector
+    @connector ||= Accounting::Quickbooks::Connector.new Division.root.custom_data
```

Comment:
I don't even know how this works right now. Seems like this should be a class var. But I'd rather avoid the class method stuff anyway unless it's needed for some reason I'm not seeing. See above.